src/registration.ts

```diff
@@ -46,20 +51,10 @@ class Registration extends Authentication {
 const user = new Registration();
 
 export function registerNewUser() {
```

Comment:
Just do your exports at the bottom of the file >_<

Reply:
moved exports at bottom of file.